jekyll/_cci1/parallel-manual-setup.md

```diff
@@ -16,11 +16,23 @@ Go to **Project Settings > Parallelism** to adjust the settings.
 
 When you use CircleCI's parallelization, we run your code on multiple separate VMs.
 To use parallelism, you make your test runner run only a subset of tests on each VM.
+
 There are two mechanisms for splitting tests among nodes: using the `files`
 configuration modifier (a very simple and straightforward way for most use cases), or
-using parallelism environment variables (aimed for  more complex scenarios).
+using parallelism environment variables (aimed at more complex scenarios).
+
+Either way, there are a few things to keep in mind:
+* `test` and `deploy` sections in your `circle.yml` are single-node only by
+  default and will only run on node 0. However, you can override `test` sections
+  per command (add `:` to the end of each command you'd like to parallelize, then
+  add a double-indented `parallel: true` below that). Alas, no parallel deploys.
```

Comment:
> Alas, no parallel deploys.
I would word this differently.

Single node deploys are intentional and a good thing.  You only want one machine pushing code to production at a time.